### ISSUE TITLE
Console dragging.

### ISSUE THREAD

**Issue by ChrisLane** [alecgorge/AdminiumIssues#63]
When dragging the console to the left, everything works fine.
It is not possible to drag the console to the right because it starts to open the menu.

**Comment by alecgorge** [comment]
I'm not seeing this one. I can scroll in every direction just fine. Do you have any other details?

**Comment by ChrisLane** [comment]
No, unfortunately not. It only seems to happen sometimes and seemingly there is is no cause.
